unabridged-works-volume-1/original/03-way-of-life.adoc

```diff
@@ -511,8 +514,15 @@ Now it may be instanced in every particular how the state of Christianity was lo
 but that would be too vast and tedious: it may suffice, therefore,
 to instance in some considerable ones, which may lead into the discovery of the rest.
 
-1+++.+++ The true rule of Christianity, or the rule of a Christian, which is to direct, guide,
-and order him in his whole course, was apostatized from, and lost.
+[.numbered-group]
+====
+
+[.numbered]
+1+++.+++ _The true rule of Christianity, or the rule of a Christian, which is to direct, guide,
+and order him in his whole course, was apostatized from, and lost._
+
+[.small-break]
```

Comment:
Okay , on this numbered group, it felt like it needed a small break between the italicized stuff and the rest of the text because of how it was really distinguished that way in the ODT. However, further down, there is another numbered group where the italicized text is not distinguished from the rest of the text and that seems fine there. So idk if that's okay for continuity and whatnot.

Reply:
I think that's fine. 👍